app/controllers/catalog_controller.rb

```diff
@@ -66,17 +66,15 @@ class CatalogController < ApplicationController
     # ------------------------------------------------------------------------------------------
     # solr fields to be displayed in the index (search results) view
     #   The ordering of the field names is the order of the display
-    config.add_index_field solr_name("repository",        :stored_sortable), label: "Library", helper_method: :render_repository_link
-    config.add_index_field solr_name("unittitle",         :displayable),  :label => "Title", :helper_method => :render_field_item
-    config.add_index_field solr_name("abstract",          :displayable),  :label => "Abstract", :helper_method => :render_field_item
     config.add_index_field solr_name("format",            :displayable),  :label => "Format", :helper_method => :render_field_item
-    config.add_index_field solr_name("language",          :displayable),  :label => "Language", :helper_method => :render_field_item
-    config.add_index_field solr_name("unitdate",          :displayable),  :label => "Dates", :helper_method => :render_field_item
-    config.add_index_field solr_name("collection",        :displayable),  :label => "Archival Collection", :helper_method => :render_collection_facet_link
-    config.add_index_field solr_name("parent_unittitles", :displayable),  :label => "Series", :helper_method => :render_series_facet_link
-    config.add_index_field solr_name("location",          :displayable),  :label => "Location", :helper_method => :render_field_item
-
-    # ------------------------------------------------------------------------------------------
+    config.add_index_field solr_name("heading",           :displayable),  :label => "Contained in", :highlight => true, :helper_method => :render_series_facet_link, unless: :is_collection?
+    config.add_index_field solr_name("unitdate",          :displayable),  :label => "Date range", :helper_method => :render_field_item
+    config.add_index_field solr_name("abstract",          :displayable),  :label => "Abstract", :highlight => true, :helper_method => :render_field_item
+    config.add_index_field solr_name("collection",        :displayable),  :label => "Archival Collection", :helper_method => :render_collection_facet_link, :highlight => true
+    config.add_index_field solr_name("repository",        :stored_sortable), label: "Library", helper_method: :render_repository_facet_link
```

Comment:
@eshadatta It looks like when you rebased you reverted one of the changes from my stable_repo_links user story. If you change this line back to render_repository_link (instead of render_repository_facet_link), most of these should turn green.